### PR TITLE
Enable to change file name

### PR DESCRIPTION
**Pull request by jkugiya**
The file name `.env` may be unfavorable due to circumstances of the project's build system.
This PR satisfies such niche requirement.

### CODE REVIEW

**Comment by mefellows** [comment]
Sorry for the delay on this, I've been travelling.

It looks OK to my eyes and given it's backwards compatible I don't see any reason why not to bring it in.

@Philippus ?

File: README.md [hunk 1]
```diff
@@ -54,6 +54,12 @@ MY_CERT="-----BEGIN CERTIFICATE-----
 
 Variable expansion of the form `$FOO` and `${FOO}` is supported based on the values in `.env` or the system environment.
 
+### Change file name
+It is possible to the file name `.env`
```
Comment (Philippus) [inline README.md]:
```suggestion
It is possible to change the file name `.env`
```

**Comment by Philippus** [comment]
I'll try to find some time and test it out.

File: README.md (hunk 1 above)
Comment (jkugiya) [reply README.md]: Added the word.

**Pull request merged**